From: Simon McVittie <smcv@debian.org>
Date: Wed, 16 Feb 2022 10:27:08 +0000
Subject: Disable web fonts for now

The way this works is still changing upstream, and getting the
correct licensing and source code for the Red Hat font (ITP: #986999)
is not really in the scope of packaging gi-docgen.

Forwarded: not-needed, Debian-specific
---
 gi_docgen.egg-info/SOURCES.txt      |  31 -------
 gidocgen/templates/basic/basic.toml |  31 -------
 gidocgen/templates/basic/fonts.css  | 176 ------------------------------------
 gidocgen/templates/basic/style.css  |   2 -
 setup.py                            |   2 -
 5 files changed, 242 deletions(-)

diff --git a/gi_docgen.egg-info/SOURCES.txt b/gi_docgen.egg-info/SOURCES.txt
index 6a30d15..3bf40bd 100644
--- a/gi_docgen.egg-info/SOURCES.txt
+++ b/gi_docgen.egg-info/SOURCES.txt
@@ -67,37 +67,6 @@ gidocgen/utils.py
 gidocgen/gir/__init__.py
 gidocgen/gir/ast.py
 gidocgen/gir/parser.py
-gidocgen/templates/basic/RedHatDisplay-Black.woff
-gidocgen/templates/basic/RedHatDisplay-Black.woff2
-gidocgen/templates/basic/RedHatDisplay-BlackItalic.woff
-gidocgen/templates/basic/RedHatDisplay-BlackItalic.woff2
-gidocgen/templates/basic/RedHatDisplay-Bold.woff
-gidocgen/templates/basic/RedHatDisplay-Bold.woff2
-gidocgen/templates/basic/RedHatDisplay-BoldItalic.woff
-gidocgen/templates/basic/RedHatDisplay-BoldItalic.woff2
-gidocgen/templates/basic/RedHatDisplay-Italic.woff
-gidocgen/templates/basic/RedHatDisplay-Italic.woff2
-gidocgen/templates/basic/RedHatDisplay-Medium.woff
-gidocgen/templates/basic/RedHatDisplay-Medium.woff2
-gidocgen/templates/basic/RedHatDisplay-MediumItalic.woff
-gidocgen/templates/basic/RedHatDisplay-MediumItalic.woff2
-gidocgen/templates/basic/RedHatDisplay-Regular.woff
-gidocgen/templates/basic/RedHatDisplay-Regular.woff2
-gidocgen/templates/basic/RedHatText-Bold.woff
-gidocgen/templates/basic/RedHatText-Bold.woff2
-gidocgen/templates/basic/RedHatText-BoldItalic.woff
-gidocgen/templates/basic/RedHatText-BoldItalic.woff2
-gidocgen/templates/basic/RedHatText-Italic.woff
-gidocgen/templates/basic/RedHatText-Italic.woff2
-gidocgen/templates/basic/RedHatText-Medium.woff
-gidocgen/templates/basic/RedHatText-Medium.woff2
-gidocgen/templates/basic/RedHatText-MediumItalic.woff
-gidocgen/templates/basic/RedHatText-MediumItalic.woff2
-gidocgen/templates/basic/RedHatText-Regular.woff
-gidocgen/templates/basic/RedHatText-Regular.woff2
-gidocgen/templates/basic/SourceCodePro-It.ttf.woff
-gidocgen/templates/basic/SourceCodePro-Regular.ttf.woff
-gidocgen/templates/basic/SourceCodePro-Semibold.ttf.woff
 gidocgen/templates/basic/base.html
 gidocgen/templates/basic/basic.toml
 gidocgen/templates/basic/class.html
diff --git a/gidocgen/templates/basic/basic.toml b/gidocgen/templates/basic/basic.toml
index 3274d04..30050a1 100644
--- a/gidocgen/templates/basic/basic.toml
+++ b/gidocgen/templates/basic/basic.toml
@@ -42,35 +42,4 @@ files = [
   "solarized-light.css",
   "solarized-dark.css",
   "search.js",
-  "RedHatDisplay-Black.woff",
-  "RedHatDisplay-Black.woff2",
-  "RedHatDisplay-BlackItalic.woff",
-  "RedHatDisplay-BlackItalic.woff2",
-  "RedHatDisplay-Bold.woff",
-  "RedHatDisplay-Bold.woff2",
-  "RedHatDisplay-BoldItalic.woff",
-  "RedHatDisplay-BoldItalic.woff2",
-  "RedHatDisplay-Italic.woff",
-  "RedHatDisplay-Italic.woff2",
-  "RedHatDisplay-Medium.woff",
-  "RedHatDisplay-Medium.woff2",
-  "RedHatDisplay-MediumItalic.woff",
-  "RedHatDisplay-MediumItalic.woff2",
-  "RedHatDisplay-Regular.woff",
-  "RedHatDisplay-Regular.woff2",
-  "RedHatText-Bold.woff",
-  "RedHatText-Bold.woff2",
-  "RedHatText-BoldItalic.woff",
-  "RedHatText-BoldItalic.woff2",
-  "RedHatText-Italic.woff",
-  "RedHatText-Italic.woff2",
-  "RedHatText-Medium.woff",
-  "RedHatText-Medium.woff2",
-  "RedHatText-MediumItalic.woff",
-  "RedHatText-MediumItalic.woff2",
-  "RedHatText-Regular.woff",
-  "RedHatText-Regular.woff2",
-  "SourceCodePro-It.ttf.woff",
-  "SourceCodePro-Regular.ttf.woff",
-  "SourceCodePro-Semibold.ttf.woff",
 ]
diff --git a/gidocgen/templates/basic/fonts.css b/gidocgen/templates/basic/fonts.css
index ce790e7..0a01b93 100644
--- a/gidocgen/templates/basic/fonts.css
+++ b/gidocgen/templates/basic/fonts.css
@@ -3,179 +3,3 @@
  *
  * SPDX-License-Identifier: Apache-2.0 OR GPL-3.0-or-later
  */
-
-/**
- * RedHat Fonts taken from https://github.com/RedHatOfficial/RedHatFont
- * License: SIL Open Font License 1.1 http://scripts.sil.org/OFL
- *
- * SourceCodePro Fonts
- * License: SIL Open Font License 1.1 https://scripts.sil.org/OFL
- */
-
-@font-face {
-  font-family: "Red Hat Display";
-  src: local('RedHatDisplayWeb'),
-       url("RedHatDisplay-Regular.woff2") format("woff2"),
-       url("RedHatDisplay-Regular.woff") format("woff");
-  font-style: normal;
-  font-weight: 400;
-  font-display: fallback;
-}
-
-@font-face {
-  font-family: "Red Hat Display";
-  src: local('RedHatDisplayWeb'),
-       url("RedHatDisplay-RegularItalic.woff2") format("woff2"),
-       url("RedHatDisplay-RegularItalic.woff") format("woff");
-  font-style: italic;
-  font-weight: 400;
-  font-display: fallback;
-}
-
-@font-face {
-  font-family: "Red Hat Display";
-  src: local('RedHatDisplayWeb'),
-       url("RedHatDisplay-Medium.woff2") format("woff2"),
-       url("RedHatDisplay-Medium.woff") format("woff");
-  font-style: normal;
-  font-weight: 500;
-  font-display: fallback;
-}
-
-@font-face {
-  font-family: "Red Hat Display";
-  src: local('RedHatDisplayWeb'),
-       url("RedHatDisplay-MediumItalic.woff2") format("woff2"),
-       url("RedHatDisplay-MediumItalic.woff") format("woff");
-  font-style: italic;
-  font-weight: 500;
-  font-display: fallback;
-}
-
-@font-face {
-  font-family: "Red Hat Display";
-  src: local('RedHatDisplayWeb'),
-       url("RedHatDisplay-Bold.woff2") format("woff2"),
-       url("RedHatDisplay-Bold.woff") format("woff");
-  font-style: normal;
-  font-weight: 700;
-  font-display: fallback;
-}
-
-@font-face {
-  font-family: "Red Hat Display";
-  src: local('RedHatDisplayWeb'),
-       url("RedHatDisplay-BoldItalic.woff2") format("woff2"),
-       url("RedHatDisplay-BoldItalic.woff") format("woff");
-  font-style: italic;
-  font-weight: 700;
-  font-display: fallback;
-}
-
-
-@font-face {
-  font-family: "Red Hat Display";
-  src: local('RedHatDisplayWeb'),
-       url("RedHatDisplay-Black.woff2") format("woff2"),
-       url("RedHatDisplay-Black.woff") format("woff");
-  font-style: normal;
-  font-weight: 900;
-  font-display: fallback;
-}
-
-@font-face {
-  font-family: "Red Hat Display";
-  src: local('RedHatDisplayWeb'),
-       url("RedHatDisplay-BlackItalic.woff2") format("woff2"),
-       url("RedHatDisplay-BlackItalic.woff") format("woff");
-  font-style: italic;
-  font-weight: 900;
-  font-display: fallback;
-}
-
-@font-face {
-  font-family: "Red Hat Text";
-  src: local('RedHatTextWeb'),
-       url("RedHatText-Regular.woff2") format("woff2"),
-       url("RedHatText-Regular.woff") format("woff");
-  font-style: normal;
-  font-weight: 400;
-  font-display: fallback;
-}
-
-@font-face {
-  font-family: "Red Hat Text";
-  src: local('RedHatTextWeb'),
-       url("RedHatText-Italic.woff2") format("woff2"),
-       url("RedHatText-Italic.woff") format("woff");
-  font-style: italic;
-  font-weight: 400;
-  font-display: fallback;
-}
-
-@font-face {
-  font-family: "Red Hat Text";
-  src: local('RedHatTextWeb'),
-       url("RedHatText-Medium.woff2") format("woff2"),
-       url("RedHatText-Medium.woff") format("woff");
-  font-style: normal;
-  font-weight: 700;
-  font-display: fallback;
-}
-
-@font-face {
-  font-family: "Red Hat Text";
-  src: local('RedHatTextWeb'),
-       url("RedHatText-MediumItalic.woff2") format("woff2"),
-       url("RedHatText-MediumItalic.woff") format("woff");
-  font-style: italic;
-  font-weight: 700;
-  font-display: fallback;
-}
-
-@font-face {
-  font-family: "Red Hat Text";
-  src: local('RedHatTextWeb'),
-       url("RedHatText-Bold.woff2") format("woff2"),
-       url("RedHatText-Bold.woff") format("woff");
-  font-style: normal;
-  font-weight: 900;
-  font-display: fallback;
-}
-
-@font-face {
-  font-family: "Red Hat Text";
-  src: local('RedHatTextWeb'),
-       url("RedHatText-BoldItalic.woff2") format("woff2"),
-       url("RedHatText-BoldItalic.woff") format("woff");
-  font-style: italic;
-  font-weight: 900;
-  font-display: fallback;
-}
-
-/* Do not fall back to locally installed SourceCodePro, as there are
- * known bad copies being distributed to this day
- */
-@font-face {
-  font-family: 'Source Code Pro';
-  font-style: normal;
-  font-weight: 400;
-  src: url("SourceCodePro-Regular.ttf.woff") format('woff');
-  font-display: swap;
-}
-
-@font-face {
-  font-family: 'Source Code Pro';
-  font-style: italic;
-  font-weight: 400;
-  src: url("SourceCodePro-It.ttf.woff") format('woff');
-  font-display: swap;
-}
-
-@font-face {
-  font-family: 'Source Code Pro';
-  font-style: normal;
-  font-weight: 600;
-  src: url("SourceCodePro-Semibold.ttf.woff") format('woff');
-  font-display: swap;
-}
diff --git a/gidocgen/templates/basic/style.css b/gidocgen/templates/basic/style.css
index 188c9a7..954b5d8 100644
--- a/gidocgen/templates/basic/style.css
+++ b/gidocgen/templates/basic/style.css
@@ -7,8 +7,6 @@
 @import url("solarized-light.css") (prefers-color-scheme: light);
 @import url("solarized-dark.css") (prefers-color-scheme: dark);
 
-@import url("fonts.css");
-
 /*********************************
  * LIGHT THEME
  *********************************/
diff --git a/setup.py b/setup.py
index e38ae79..b18ac25 100644
--- a/setup.py
+++ b/setup.py
@@ -49,8 +49,6 @@ package_data = {
         "templates/basic/*.html",
         "templates/basic/*.js",
         "templates/basic/*.png",
-        "templates/basic/*.woff2",
-        "templates/basic/*.woff",
     ],
 }
 
